src/test/resources/org/rust/ide/annotator/fixtures/attributes.rs

```diff
@@ -1,4 +1,4 @@
 <info>#[cfg_attr(foo)]</info>
 fn <info>main</info>() {
-    <info>#![crate_type = "lib"]</info>
+    <info>#![crate_type = <info>"lib"</info>]</info>
```

Comment:
We have such annotation layout:
```
|--------------|
         |--|
```

While if nested tags weren't supported, test framework should (as far as I understand) expect this:
```
|---------|
       |----------|
```